zebra/rt_netlink.c

```diff
@@ -1457,38 +1459,44 @@ static bool _netlink_route_encode_nexthop_src(const struct nexthop *nexthop,
 	return true;
 }
 
+/* Define the maximum number of SIDs in the segment list */
+#define SRV6_MAX_SEGS 4
```

Comment:
Isn't this in lib/srv6.h?  How are we going to keep these two values inline with each other?

Reply:
Yes, Done. This #define is only in srv6.h